dev/com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.1.envVars/build.gradle

```diff
@@ -0,0 +1,79 @@
+/*******************************************************************************
+ * Copyright (c) 2018,2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+dependencies {
```

Comment:
Similar comment about Gradle dependencies as the bnd buildpath dependencies - I bet there are several that could be taken out from multiple FAT projects.

Reply:
Opened https://github.com/OpenLiberty/open-liberty/issues/23549 to address this